esim_torch/esim_torch.py

```diff
@@ -3,9 +3,13 @@
 
 
 class EventSimulator_torch(torch.nn.Module):
-    def __init__(self, contrast_threshold):
-        self.contrast_threshold = contrast_threshold
+    def __init__(self, contrast_threshold_neg=0.2, contrast_threshold_pos=0.2, refractory_period=0):
```

Comment:
is `refractory_period` in nanoseconds? If yes, I would rename it to `refractory_period_ns` to make it obvious

Reply:
done